sp800-63b/sec4_aal.md

```diff
@@ -192,10 +196,8 @@ Requirement | AAL 1 | AAL 2 | AAL 3
 **FIPS 140 verification** | Level 1 (Government agency verifiers) | Level 1 (Government agency authenticators and verifiers) | Level 2 overall (MF authenticators)<br />Level 1 overall (verifiers and SF Crypto Devices)<br />Level 3 physical security (all authenticators)
 **Reauthentication** | 30 days | 12 hours or 30 minutes inactivity; may use one authentication factor | 12 hours or 15 minutes inactivity; shall use both authentication factors
 **Security controls**|[[SP 800-53]](#SP800-53) Low Baseline (or equivalent)|[[SP 800-53]](#SP800-53) Moderate Baseline (or equivalent)|[[SP 800-53]](#SP800-53) High Baseline (or equivalent)
-**Privacy controls**|[[SP 800-53]](#SP800-53)(or equivalent)|[[SP 800-53]](#SP800-53) (or equivalent)|[[SP 800-53]](#SP800-53) (or equivalent)
 **MITM resistance** | Required | Required | Required |
 **Verifier impersonation resistance** | Not required | Not required | Required |
```

Comment:
Having it be just informational isn't actionable by anyone. I'd like to throw it in as a requirement because that is what will prompt feedback. But let's merge this first and I'll rebase that change on top of this.